common/cards/default/hermits/geminitay-rare.ts

```diff
@@ -32,27 +32,32 @@ class GeminiTayRareHermitCard extends HermitCard {
 		const {player} = pos
 
 		// @TODO egg confusion, and how can we get rid of follow up
+		// The follow-up is now present for the counterpart.
 		// is that even in the scope of this refactor?
-		player.hooks.afterAttack.add(instance, (attack) => {
+		player.hooks.onAttack.add(instance, (attack) => {
```

Comment:
Why does this need to be changed for a hermit card?

Reply:
So she wouldn't delete Knockback before it is applied but she would still delete it.

Reply:
This won't work, completedActions are not yet set in onAttack and therefore can't yet be modified. I'm also still not sure exactly what you're fixing by changing to onAttack here

Reply:
This does modify `completedActions` during `afterAttack`, the hook must be added here so it is executed after Knockback. Otherwise Knockback will 'apply' after Geminislay, since hermits are attached before single-use effects and Knockback shouldn't _need_ first priority in `afterAttack`.